docker/testrunner/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
```

Comment:
It is recommended to run both `apt-get` commands in a single `RUN` statement:

```
RUN apt-get update && apt-get install -y \
        openjdk-7-jdk \
        sudo \
        xorg \
        xserver-xorg-video-dummy \
        xvfb
```

Reference: https://docs.docker.com/engine/articles/dockerfile_best-practices/#apt-get